index.html

```diff
@@ -958,7 +940,7 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
       Timing-Allow-Origin = 1#( <a data-cite=
 "FETCH#origin-header">origin-or-null</a> / <a data-cite=
 "FETCH#http-new-header-syntax">wildcard</a> )
-    </pre>
+</pre>
```

Comment:
In my preview this is showing the content has changed to a bunch of lines instead of a single line

Reply:
I reverted that change and still see it in the diff... Preview seems fine though